expat/lib/xmlparse.c

```diff
@@ -2544,13 +2557,15 @@ XML_GetFeatureList(void) {
      XML_L("XML_BLAP_ACT_THRES"),
      EXPAT_BILLION_LAUGHS_ATTACK_PROTECTION_ACTIVATION_THRESHOLD_DEFAULT},
 #endif
+    /* Added in Expat 2.6.0. */
+    {XML_FEATURE_DTD, XML_L("XML_GE"), XML_GE * 2 - 1},
```

Comment:
This appears to be the first time multiple entries in this array have the same "feature" value. Is there a risk that this could cause confusion for users, or do we expect them to make a comparison with the "name" string?

Reply:
That's a great catch, I was mindlessly copy-pasting here apparently, re-use of `XML_FEATURE_DTD` was not intended at all. Added `XML_FEATURE_GE` now. Thanks! :pray: